static/page-templates/page-templates.json

```diff
@@ -89,6 +94,10 @@
 		{
 			"title": "maywood",
 			"demo": "https://maywooddemo.wordpress.com/"
+		},
+		{
+			"title": "barnsbury",
+			"demo": "https://barnsburydemo.wordpress.com/"
```

Comment:
What the point of this array? Does the `public-api.wordpress.com/rest/v1/template/demo` API need it to generate the demo?

Reply:
Apologies for the 28-days-late answer to this question, but that API endpoint doesn't use this file. I don't _think_ the file is currently used anywhere at all, actually! I wonder if this file was a precursor to what we now have in `available-designs-config.json`? For consistency though, I don't see an issue with adding them here, and eventually we'll want to have an endpoint that can list info similar to this, I suppose...

Reply:
> I don't think the file is currently used anywhere at all, actually! 

Thanks! That's what I was thinking! I just wanted to make sure I wasn't missing something again. 🥴

Reply:
I don't think it is used anywhere neither.

@johnHackworth remember details?

Reply:
I just came across it yesterday. It's used by the verticals MC tool only. :)

There might be scope to simplify/merge these config files. We're moving them over to the server for now, mainly to reduce the number of remote gets we're doing in our APIs, but also since devs are the only ones updating them anyway.

Reply:
> There might be scope to simplify/merge these config files.

Yep, see https://github.com/Automattic/wp-calypso/pull/40587